acceptance/setup/common/pre-suite/010_install_ruby.rb

```diff
@@ -18,6 +18,8 @@
         version = /ruby (2\.[0-9])/.match(output.stdout)[1].delete('.')
         bolt.add_env_var('PATH', "/cygdrive/c/tools/ruby#{version}/bin:PATH")
       end
+      # public_suffix for win errored with version > 4.0.7
```

Comment:
I think you should note the incompatible version of ruby that creates this incompatability so that if we ever bump the ruby version we can re-assess if we need to continue pinning.